### PR TITLE
Cloudflare AI integration

### PR DESCRIPTION
**Pull request by anonthedev**
Added Cloudflare integration in the sdk

### CODE REVIEW

File: js/src/sdk/base.toolset.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    return g = { next: verb(0), "throw": verb(1), "return": verb(2) }, typeof Symbol === "function" && (g[Symbol.iterator] = function() { return this; }), g;
+    function verb(n) { return function (v) { return step([n, v]); }; }
+    function step(op) {
+        if (f) throw new TypeError("Generator is already executing.");
+        while (_) try {
+            if (f = 1, y && (t = op[0] & 2 ? y["return"] : op[0] ? y["throw"] || ((t = y["return"]) && t.call(y), 0) : y.next) && !(t = t.call(y, op[1])).done) return t;
+            if (y = 0, t) op = [op[0] & 2, t.value];
+            switch (op[0]) {
+                case 0: case 1: t = op; break;
+                case 4: _.label++; return { value: op[1], done: false };
```
Comment (himanshu-dixit) [inline js/src/sdk/base.toolset.js:22]: Remove these files

File: js/examples/ai-playground-main/package-lock.json [hunk 2]
```diff
@@ -0,0 +1,1467 @@
...
+      "version": "0.0.0",
+      "dependencies": {
+        "@faker-js/faker": "^8.4.1",
+        "hono": "^4.2.5"
+      },
+      "devDependencies": {
+        "@cloudflare/workers-types": "^4.20230419.0",
+        "typescript": "^5.0.4",
+        "wrangler": "^3.0.0"
+      }
```
Comment (himanshu-dixit) [inline js/examples/ai-playground-main/package-lock.json:18]: Remove this

**Review by himanshu-dixit — CHANGES_REQUESTED** [review]
Remove js files and package-lock

**Comment by utkarsh-dixit** [comment]
### Pull Request Summary

**Describe the changes of this PR and its objective:**
This PR introduces a new feature integrating Cloudflare framework with ComposioCore to support AI functionality. It adds necessary configuration, dependencies, and examples to demonstrate AI-powered issue creation on GitHub using Cloudflare Workers. 

**Objective:**
- Implement and demonstrate an AI-powered playground using Cloudflare Workers.
- Add support for `Cloudflare` in the core composio library.
- Provide examples and configurations for deploying and testing the Cloudflare-based solution.

**Categorize this PR:**
- **Type:** Feature

**Important change files containing the major logical changes:**
1. `js/examples/ai-playground-main/src/worker.js`
2. `js/src/frameworks/cloudflare.ts`
3. `js/src/index.ts`
4. `js/src/sdk/base.toolset.ts`
5. `js/examples/ai-playground-main/README.md`
6. `js/examples/ai-playground-main/package.json`
7. `js/webpack.config.js`
8. `js/wrangler.toml`

**Code Changes Summary:**
- **New Example:** Added `ai-playground-main` directory containing example Cloudflare worker implementation.
- **Wrangler Config:** Added `wrangler.toml` to configure Cloudflare worker.
- **Dependency Updates:** Added new dependencies such as `composio-core`, `hono`, and `wrangler`.
- **Cloudflare Integration:** Created methods for Cloudflare-specific actions in `cloudflare.ts`.

**Review by himanshu-dixit — APPROVED** [review]

**Pull request merged**